azure/posix.yml

```diff
@@ -73,3 +73,11 @@ jobs:
 
           # install_run $PLAT
         displayName: Build, install, and test
+      - script: pip install twine
+        displayName: Install twine
+      - task: TwineAuthenticate@0
+        inputs:
+          artifactFeeds: "sklearn-nightly"
```

Comment:
I get that this is for testing for now but just to note that for the final config, for the release builds we should probably name the feed "release" or "sklearn-release".